components/console/introduction.rst

```diff
@@ -124,6 +115,14 @@ This prints::
 Coloring the Output
 ~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+    By default, the Windows command console doesn't support output coloring. The
+    Console component disables output coloring for Windows systems, but if your
+    commands invoke other scripts which emit color sequences, they will be
+    wrongly displayed as raw escape characters. Install the free `ANSICON`_
```

Comment:
The note should mention both ANSICON and ConEmu.

ConEmu might be a better solution btw, because ANSICON is flagged as malware by several antivirus tools because of the way it works (it intercepts the CMD buffer). ConEmu does not suffer from this, because it is a terminal on its own (embedding any shell in it, be it CMD, Powershell, GitBash, etc...)